packages/sdk/src/types/internal/context.ts

```diff
@@ -309,7 +309,7 @@ export class InternalContext {
 			execution.on('protocol.destroy-actors', this.localDestroyActors);
 			execution.on('protocol.user-joined', this.userJoined);
 			execution.on('protocol.user-left', this.userLeft);
-			execution.on('protocol.update-user', this.updateUser);
+			execution.on('protocol.user-update', this.updateUser);

```

Comment:
>user-update [](start = 26, length = 11)

The user-update system isn't in use, and will need revamping if/when it does start getting used, but I'm not sure the changes you made here are an improvement over what was there already. I'd feel better these changes were reverted.

Reply:
fixed